go.mod

```diff
@@ -86,6 +86,7 @@ replace (
 	// fix build issue https://github.com/docker/distribution/issues/2406
 	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
+	github.com/oam-dev/terraform-controller v0.1.4 => github.com/zzxwill/terraform-controller v0.1.5
```

Comment:
Will remove it after https://github.com/oam-dev/terraform-controller/pull/18 is merged.